### ISSUE TITLE
Custom views: The loading spinner for showing children seems to take longer than before

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#76999]
Issue Type: <b>Bug</b>

Since the change to the new tree, when expanding a node with children loaded via a promise, it seems to take the loading spinner longer to appear than before. And the progress indicator at the top of the sidebar will show up before the loading spinner -- and if the delay isn't long enough you could see the sidebar progress indicator and never see the node loading spinner.

//cc @alexr00 

VS Code version: Code - Insiders 1.37.0-insider (9296aaa2201b638e2b1dc71a9b9fe38c5ab2e904, 2019-07-09T05:26:27.407Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4008)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.93GB (17.89GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by alexr00** [comment]
I went back and tested this with the old tree. The progress indicator showing up before the loading icon on the viewlet icon happened before too. If the delay isn't long enough, it only shows at the top of the viewlet, and you won't see the icon on the viewlet icon.

The icon showing lives outside of the code that was changed with the custom tree view tree update, so I wouldn't expect that timing to have changed.
